nopol/src/main/java/fr/inria/lille/spirals/repair/synthesizer/collect/spoon/DefaultConstantCollector.java

```diff
@@ -0,0 +1,58 @@
+package fr.inria.lille.spirals.repair.synthesizer.collect.spoon;
+
+import spoon.processing.AbstractProcessor;
+import spoon.reflect.code.CtAssignment;
+import spoon.reflect.code.CtLiteral;
+import spoon.reflect.code.CtLocalVariable;
+import spoon.reflect.code.CtThrow;
+import spoon.reflect.declaration.CtElement;
+import spoon.reflect.declaration.CtField;
+import spoon.reflect.declaration.CtMethod;
+
+/**
+ * Created by bdanglot on 10/4/16.
+ */
+public class DefaultConstantCollector extends AbstractProcessor<CtLiteral> {
+
```

Comment:
This class has no impact on the state of the program (contain only some checks and return statement). We should just remove it IMHO.